inputfiles/browser.webidl.preprocessed.json

```diff
@@ -74396,6 +74396,13 @@
                             "type": "DOMString",
                             "content-attribute-reflects": 1
                         },
+                        "sheet": {
```

Comment:
This is a generated file too, so this change should go in addedTypes.json.

(The format there is slightly different there, I think, unfortunately.)